.github/actions/sync-fork-with-upstream/action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Sync sources'
+author: 'Apache Spark'
+description: 'A composite GitHub Action that syncs fork branch with Spark master'
+
+runs:
+  using: 'composite'
+  steps:
+  - name: Sync the current branch with the latest master in Apache Spark
+    if: github.repository != 'apache/spark'
+    shell: bash
+    run: |
+      echo "APACHE_SPARK_REF=$(git rev-parse HEAD)" >> $GITHUB_ENV
+      git fetch https://github.com/$GITHUB_REPOSITORY.git ${GITHUB_REF#refs/heads/}
+      git -c user.name='Apache Spark Test Account' -c user.email='sparktestacc@gmail.com' merge --no-commit --progress --squash FETCH_HEAD
+      git -c user.name='Apache Spark Test Account' -c user.email='sparktestacc@gmail.com' commit -m "Merged commit" --allow-empty
+
+branding:
+  icon: 'check-circle'
```

Comment:
quick question: what is this?

Reply:
This is defines the icon of the action for UI and marketplace purposes, for instance the blue circle with the triangle at https://github.com/marketplace/actions/checkout

Not sure if this is required, does not make much sense if action is not published.